tests/unit/conftest.py

```diff
@@ -521,6 +522,29 @@ def extra_token(app, db, oauth2_client, users):
     )
 
 
+@pytest.fixture
+def rat_generate_token(app, db, oauth2_client, users):
+    """Create token."""
+    with db.session.begin_nested():
+        token_ = Token(
+            client_id=oauth2_client,
+            user_id=users[0]['id'],
+            access_token='rat_token',
+            expires=datetime.utcnow() + timedelta(hours=10),
+            is_personal=False,
+            is_internal=True,
+            _scopes=tokens_generate_scope.id,
+        )
+        db.session.add(token_)
+    db.session.commit()
+    return dict(
+        token=token_,
+        auth_header=[
```

Comment:
(Nitpicking)It's not clear to me why you are returning this header here, since you are not using it and it can still be constructed with the returned object.